### PR TITLE
test: fixing gating transactions tests

### PR DESCRIPTION
**Pull request by amelnytskyi**
# What ❔

Fixed disabled (xdescribe and xit) /transactions/{transactionHash}/transfers tests and turned them on back 

## Why ❔

/transactions/{transactionHash}/transfers tests were disabled since they were outdated

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.

### CODE REVIEW

File: packages/integration-tests/tests/api/transactions.test.ts [hunk 1]
```diff
@@ -109,736 +109,1071 @@ describe("Transactions", () => {
...
-        txHash = await helper.getStringFromFile(bufferFile + Buffer.txEthWithdrawOtherAddress);
-        apiRoute = `/transactions/${txHash}`;
-        response = await helper.performGETrequest(apiRoute);
-
-        expect(response.status).toBe(200);
-        expect(response.body.hash).toBe(txHash);
-        expect(response.body.to).toBe("0x000000000000000000000000000000000000800A");
-        expect(response.body.from).toBe(Wallets.richWalletAddress);
-        expect(response.body.value).toBe("9000000000000");
+    describe("/transactions/{transactionHash}", () => {
```
Comment (pcheremu) [inline packages/integration-tests/tests/api/transactions.test.ts:247]: Do we really need this tab? Do we have describe("/transactions/{transactionHash}" inside of describe("/transactions/{transactionHash}/transfers" ?

Comment (amelnytskyi) [reply packages/integration-tests/tests/api/transactions.test.ts]: Fixed

File: packages/integration-tests/tests/api/transactions.test.ts [hunk 2]
```diff
@@ -381,7 +462,20 @@ describe("Transactions", () => {
         );
```
Comment (pcheremu) [inline packages/integration-tests/tests/api/transactions.test.ts:462]: Probably some fields missed?
`
"items": [
        {
            "from": "0x3c0FD3279c2b714141aa731519b68Bd428454073",
            "to": "0x0000000000000000000000000000000000008001",
            "blockNumber": 16511607,
            "transactionHash": "0xbe1334c29408d1fe90a810fbda46da059f18cef033bfb08d75050c531b5c789e",
            "timestamp": "2024-02-06T10:44:38.831Z",
            "amount": "1503311400000000",
            "tokenAddress": "0x000000000000000000000000000000000000800A",
            "type": "fee",
            "tokenType": "ETH",
            "fields": null,
            "isInternal": false,
            "token": {
                "l2Address": "0x000000000000000000000000000000000000800A",
                "l1Address": "0x0000000000000000000000000000000000000000",
                "symbol": "ETH",
                "name": "Ether",
                "decimals": 18,
                "usdPrice": 2294.85,
                "liquidity": 275206159445,
                "iconURL": "https://assets.coingecko.com/coins/images/279/large/ethereum.png?1696501628"
            }
        },
`
https://[block-explorer-api.testnets.zksync.dev/transactions/0xbe1334c29408d1fe90a810fbda46da059f18cef033bfb08d75050c531b5c789e/transfers](https://block-explorer-api.testnets.zksync.dev/transactions/0xbe1334c29408d1fe90a810fbda46da059f18cef033bfb08d75050c531b5c789e/transfers)

File: packages/integration-tests/tests/api/transactions.test.ts [hunk 3]
```diff
@@ -399,7 +506,20 @@ describe("Transactions", () => {
...
+              l2Address: Token.ETHER_ERC20_Address,
+              liquidity: 220000000000,
+              symbol: "ETH",
+              name: "Ether",
+              usdPrice: 1800,
+            },
+          })
+        );
       });
     });
```
Comment (pcheremu) [inline packages/integration-tests/tests/api/transactions.test.ts:525]: on 534, do we need a token and blocknumber data?

**Review by pcheremu — COMMENTED** [review]
Fixes itself looks good, I just see that some tests outdated or some fields are missed. It will be good to keep them up-to date together with its fixes. AFAK it is not necessary to be so detailed for items [1] or [2] but we have a full list of items for the item [0] if any.

**Pull request merged**